docs/code-samples/settlement/daml/Scripts/Transfer.daml

```diff
@@ -52,164 +22,100 @@ data SetupTransferRequest = SetupTransferRequest
-      issuer = bank
-      id = Id "USD"
-      version = "0"
-      description = "Instrument representing units of USD"
-      validAsOf = now
-      observers = M.empty
-    instrument = Instrument.getKey $ toInterface cashInstrument
-  cashInstrumentCid <- toInterfaceContractId @Instrument.I <$> submit bank do
-    createCmd cashInstrument
-
-  -- Alice deposits cash at the bank
-  aliceRequestCid <- submit alice do
-    createCmd CreditAccount.Request with
-      account = aliceAccount
-      instrument
-      amount = 1000.0
-  aliceCashHoldingCid <- fromInterfaceContractId @Transferable.I <$> submit bank do
-    exerciseCmd aliceRequestCid CreditAccount.Accept
+  Setup{bank1 = bank, ..} <- runSetup accountControllers
 
   -- SETUP_TRANSFER_REQUEST_BEGIN
   let
     transferRequest = Transfer.Request with
       requestor
       receiverAccount = bobAccount
-      transferableCid = coerceContractId aliceCashHoldingCid
+      transferableCid = aliceHoldingCid
       accepted = S.fromList []
       observers = S.fromList [alice, bob, bank]
   transferRequestCid <- submit requestor do createCmd transferRequest
   -- SETUP_TRANSFER_REQUEST_END
 
-  pure SetupTransferRequest with ..
+  pure SetupTransferRequest{..}
 
--- | Test scripts for testing various required authorizations for transfers.
--- 1. Dual-control transfer: The Bank and Alice can jointly transfer Alice's holding to Bob.
+-- Test scripts for testing various required authorizations for transfers.
+-- | 1. Dual-control transfer: Bank and Alice can jointly transfer Alice's holding to Bob.
 runDualControlTransfer : Script ()
 runDualControlTransfer = do
+  -- SETUP_DUAL_CONTROL_TRANSFER_BEGIN
   let dualControl = AccountControllers with incoming = Anyone; outgoing = Both
-  setupState@SetupTransferRequest{..} <- runSetupTransferRequestWith dualControl
-  -- the bank and the new owner can't effectuate the transfer.
+  setupState@SetupTransferRequest{bank, alice, bob, requestor, transferRequestCid} <-
+    runSetupTransferRequestWith dualControl
+  -- SETUP_DUAL_CONTROL_TRANSFER_END
+  -- The bank and the new owner can't effectuate the transfer.
   submitMultiMustFail [bank, bob] [] do
     exerciseCmd transferRequestCid Transfer.Effectuate with actors = S.fromList [bank, bob]
+  -- Alice and Bank can effectuate the transfer.
+  -- DUAL_CONTROL_TRANSFER_BEGIN
   transferRequestCid <- submit bank do
     exerciseCmd transferRequestCid Transfer.Accept with actors = S.singleton bank
   submit alice do
     exerciseCmd transferRequestCid Transfer.Accept with actors = S.singleton alice
+  -- DUAL_CONTROL_TRANSFER_END
   pure ()
 
--- 2. Discretionary transfer: The Bank can transfer Alice's holding to Bob.
+-- | 2. Discretionary transfer: Bank can transfer Alice's holding to Bob.
 runDiscretionaryTransfer : Script ()
 runDiscretionaryTransfer = do
+  -- SETUP_DISCRETIONARY_TRANSFER_BEGIN
   let discretionary = AccountControllers with incoming = Custodian; outgoing = Custodian
```

Comment:
Given that this gets printed in the docs, should we rather do

```suggestion
  let discretionary = AccountControllers
    with
      incoming = Custodian
      outgoing = Custodian
```

for improved readability?